core-services/image-pruner/README.md

```diff
@@ -0,0 +1,5 @@
+# Image Pruner
+
+Configuration supporting the [image pruning job](https://prow.svc.ci.openshift.org/?job=periodic-ci-image-pruner).
+
+TODO: Could use some background. Why are we pruning images?
```

Comment:
We create some ludicrous amount of images in every image build for the build farm, if we don't prune them we're just using up terabytes of space holding layers nobody will ever reference again.